### PR TITLE
Move npm build to github actions

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build
```
Comment (skjnldsv) [inline .github/workflows/nodejs.yml]: Let's e consistent on the steps case :see_no_evil: 
```suggestion
    - name: Install dependencies & build
```

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      run: |
+        npm ci
+        npm run build --if-present
+    - name: check webpack build
```
Comment (skjnldsv) [inline .github/workflows/nodejs.yml]:
```suggestion
    - name: Check webpack build changes
```

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
```
Comment (skjnldsv) [inline .github/workflows/nodejs.yml]:
```suggestion
    - name: Use node ${{ matrix.node-version }}
```

File: .github/workflows/nodejs.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+name: Node CI
```
Comment (skjnldsv) [inline .github/workflows/nodejs.yml]:
```suggestion
name: Node
```

Since this is visible in the ci list at the bottom, extra unneeded text ;)

**Review by skjnldsv — CHANGES_REQUESTED** [review]
see comments ;)

**Review by skjnldsv — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**